### PR TITLE
[CP-1498] handle core logic

### PR DESCRIPTION
**Pull request by wojcik92michal**
Jira: [CP-1498]

This PR:
* adds core logic for seeding data for e2e tests - but this is not over, more changes in the upcoming PR
* adds validator for file path for wdio (as it turned out, it does not report an error when the test file path does not exist)
* adds some improvements for e2e module (e.g. for package.json and tsconfig) 

**Description**

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

[CP-1498]: https://appnroll.atlassian.net/browse/CP-1498?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/e2e/README.md [hunk 1]
```diff
@@ -5,3 +5,7 @@ The E2E testing for Mudita Center application
 ## Usage
 
 `npm run test`
+
+# Setup info
+
+https://appnroll.atlassian.net/wiki/spaces/CP/pages/1681096705/E2E+test+environment+setup+execution+guide+v2
```
Comment (dkarski) [inline packages/e2e/README.md:11]: 👌

File: packages/e2e/src/pure/readme.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Why this module was added?
+
+Importing `@mudita/pure` package to the project and using it with ts-node (that is used by WDIO) causes unexpected problems (`Error [ERR_REQUIRE_ESM]: Must use import to load ES Module:`), mainly by `p-queue` (it uses ESM modules) . One of the solutions to this problem is to downgrade the lib, which is not the best, the other is to adjust the tsconfig/package.json configuration.
+
+To not waste time, the required code from `@mudita/pure` was just copied to this folder, as it is only few types and one class. If you have time or knowledge, feel free to fix this problem and adjust necessary changes.
```
Comment (dkarski) [inline packages/e2e/src/pure/readme.md:5]: What do you think to create issue in our backlog to fix this workround? 🤔

**Review by wercix — APPROVED** [review]

File: packages/e2e/src/seeds/main.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    seedResult = result
+  }
+}
+
+// TODO [mw] implement it - scope of the next PR
+const getData = (specFileName: string): SeedParams | undefined => {
+  return
+}
+
+export const afterDataSeed = async () => {
```
Comment (dkarski) [inline packages/e2e/src/seeds/main.ts:28]:
```suggestion
export const removeDataSeed = async () => {
```

The current function name, no descibe what the function do but why is used 😉

**Review by dkarski — APPROVED** [review]

**Review by igorbezsmertnyi — DISMISSED** [review]

File: packages/e2e/src/pure/readme.md (hunk 2 above)
Comment (wojcik92michal) [reply packages/e2e/src/pure/readme.md]: Sure, makes sense. Also linked this document with Jira task in https://github.com/mudita/mudita-center/pull/943/commits/4a4272d336144cff5c017837e28bc8ec5073c270

File: packages/e2e/src/seeds/main.ts (hunk 3 above)
Comment (wojcik92michal) [reply packages/e2e/src/seeds/main.ts]: Makes sense. Changed in https://github.com/mudita/mudita-center/pull/943/commits/e3ce4e2764516dcce32543af53f0d81508f1351a

**Review by dkarski — APPROVED** [review]

**Review by wercix — APPROVED** [review]

**Pull request merged**